.sail/Dockerfile

```diff
@@ -8,7 +9,13 @@ ENV GO111MODULE=off
 
 # Install the latest version of Hugo.
 RUN wget -O /tmp/hugo.deb https://github.com/gohugoio/hugo/releases/download/v0.55.4/hugo_extended_0.55.4_Linux-64bit.deb && \
-    sudo dpkg -i /tmp/hugo.deb && \
-    rm -f /tmp/hugo.deb
+  sudo dpkg -i /tmp/hugo.deb && \
+  rm -f /tmp/hugo.deb
 
 RUN installext peterjausovec.vscode-docker
+
+RUN sudo apt-get install curl -y
+
+RUN sudo apt-get install software-properties-common -y && \
```

Comment:
I think we should use `nvm` for installing node, see instructions [here](https://www.digitalocean.com/community/tutorials/how-to-install-node-js-on-ubuntu-18-04#installing-using-nvm).

@kylecarbs what do you think?